src/cache.cpp

```diff
@@ -435,7 +432,15 @@ void Cache::populate_tables()
 			"for version %u.%u",
 			patch_version.major,
 			patch_version.minor);
+
+		std::string update_metadata_query = "UPDATE metadata SET db_schema_version_major = " +
+			std::to_string(patch_version.major) + ", db_schema_version_minor = " +
+			std::to_string(patch_version.minor) + ";";
+
 		for (const auto& query : patches_it->second) {
+			if (patch_version > SchemaVersion{2, 11}) {
+				run_sql_nothrow(update_metadata_query);
```

Comment:
I think this only needs to be run once per set of SQL queries, so just outside of this inner for-loop.

Reply:
@dennisschagt that's right